### ISSUE TITLE
Limit causes a crash when cloud is deployed in heroku only.

### ISSUE THREAD

**Issue by AdamMagaluk** [zettajs/zetta#258]
When peering to heroku `zetta-v1-beta.herokuapp.com` with the latest code deployed.

Subscribing to heroku with `{"type": "subscribe", "topic": "**", "limit": 100 }`, when the limit is reached heroku throws an error and crashes. Doing this locally with two zetta instances does not do this.

```
2015-11-09T16:51:50.739948+00:00 app[web.1]: /app/node_modules/zetta/node_modules/spdy/lib/spdy/connection.js:630
2015-11-09T16:51:50.739953+00:00 app[web.1]:   state.framer.goawayFrame(state.lastId,
2015-11-09T16:51:50.739956+00:00 app[web.1]:
2015-11-09T16:51:50.739955+00:00 app[web.1]:          ^
2015-11-09T16:51:50.739956+00:00 app[web.1]: TypeError: invalid data
2015-11-09T16:51:50.739957+00:00 app[web.1]:     at Socket.write (net.js:617:11)
2015-11-09T16:51:50.739959+00:00 app[web.1]:     at Connection.write (/app/node_modules/zetta/node_modules/spdy/lib/spdy/connection.js:515:36)
2015-11-09T16:51:50.739960+00:00 app[web.1]:     at Scheduler.tickListener [as _tickListener] (/app/node_modules/zetta/node_modules/spdy/lib/spdy/scheduler.js:67:23)
2015-11-09T16:51:50.739960+00:00 app[web.1]:     at processImmediate [as _immediateCallback] (timers.js:383:17)
```

**Comment by AdamMagaluk** [comment]
This is a result of using node >= v4.1.2.

This is exactly the same issue we're seeing in our tests. Added issue for node support. #259

**Comment by wooliet** [comment]
Instead of updating fornode version support, would it be a good idea (for now) to specify the node version in your [package.json](https://docs.npmjs.com/files/package.json#engines)?

**Comment by AdamMagaluk** [comment]
@wooliet Yes that's probably a good idea if the issue persists for a while, I'd like to get support for 4.1.2 before we release v1.

**Comment by AdamMagaluk** [comment]
Fixed in #269
